sql/core/src/test/scala/org/apache/spark/sql/FilterPushdownBenchmark.scala

```diff
@@ -32,14 +32,14 @@ import org.apache.spark.util.{Benchmark, Utils}
  */
 object FilterPushdownBenchmark {
   val conf = new SparkConf()
-  conf.set("orc.compression", "snappy")
-  conf.set("spark.sql.parquet.compression.codec", "snappy")
+    .setMaster("local[1]")
```

Comment:
I think you can do `.setIfMissing("spark.master", "local[1]")`
that way perhaps we could get this to run on different backends too

Reply:
ok